lib/paper_trail/has_paper_trail.rb

```diff
@@ -57,7 +57,8 @@ module ClassMethods
       #
       # @api public
       def has_paper_trail(options = {})
-        paper_trail.setup(options)
+        # Merge defaults with per-model specifications
```

Comment:
Instead of having this comment here, please write a short paragraph about this in the method documentation above.

Reply:
Difference in style, but that's ok - will do